bcda/auth/alpha.go

```diff
@@ -148,12 +150,27 @@ func (p AlphaAuthPlugin) RevokeClientCredentials(clientID string) error {
 	return nil
 }
 
-// Verify credentials (client ID and secret) and return access token
-func (p AlphaAuthPlugin) GetAccessToken(creds Credentials) (Token, error) {
-	return Token{}, fmt.Errorf("GetAccessToken not implemented in Alpha")
+// MakeAccessToken manufactures an access token for the given credentials
+func (p AlphaAuthPlugin) MakeAccessToken(credentials Credentials) (string, error) {
+	if credentials.ClientSecret == "" || credentials.ClientID == "" {
+		return "", fmt.Errorf("missing or incomplete credentials")
+	}
+	aco, err := getACOByClientID(credentials.ClientID)
```

Comment:
does this func exist?  or am i missing something?

Reply:
oh, it's defined in okta.go ...  how does this work?

Reply:
Yes. It's in auth/middleware.go.

Reply:
oh ok so its defined right here in okta.go :https://github.com/CMSgov/bcda-app/blob/d76da634cc9c77ae1386057b0690c7f2e773d7f2/bcda/auth/okta.go#L176

it doesn't need to be exported because we're using it in the same package.  should we put it in a more "shared" place since it is used by alpha _and_ okta?